.github/workflows/release.yaml

```diff
@@ -331,23 +331,59 @@ jobs:
     needs: cargo-registry-cache
     runs-on: ubuntu-20.04
     timeout-minutes: 30
+    strategy:
+      matrix:
+        wasm_feature: [none, threads]
```

Comment:
changing `threads` to `parallel` yes. I'm not sure we want `wasm_threads` because in the future with simd we will probably want `none, threads, threads_simd`, I'm not sure if there are browser with simd but not shared buffer. otherwise will make sense to have `wasm_threads` and `wasm_simd`.

Reply:
according to the [wasm features table](https://webassembly.org/roadmap/) there are some cases where simd is available but threads are not. i guess it makes sense to split it into `wasm_threads: [sequential, parallel]` and later add a `wasm_simd: [single, multiple]` (or sth similar) which can then be combined in all possible configurations.